<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>4 Reproducible Research | Portfolio</title>
  <meta name="description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  <meta name="generator" content="bookdown 0.41 and GitBook 2.6.7" />

  <meta property="og:title" content="4 Reproducible Research | Portfolio" />
  <meta property="og:type" content="book" />
  
  <meta property="og:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="4 Reproducible Research | Portfolio" />
  
  <meta name="twitter:description" content="This is a minimal example of using the bookdown package to write a book. The output format for this example is bookdown::gitbook." />
  

<meta name="author" content="Rachelle Balgit" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="guerilla-analytics.html"/>
<link rel="next" href="new-packages.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/core-js-2.5.3/shim.min.js"></script>
<script src="libs/react-18.2.0/react.min.js"></script>
<script src="libs/react-18.2.0/react-dom.min.js"></script>
<script src="libs/reactwidget-2.0.0/react-tools.umd.cjs"></script>
<link href="libs/htmltools-fill-0.5.8.1/fill.css" rel="stylesheet" />
<script src="libs/htmlwidgets-1.6.4/htmlwidgets.js"></script>
<link href="libs/reactable-0.4.4/reactable.css" rel="stylesheet" />
<script src="libs/reactable-binding-0.4.4/reactable.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { display: inline-block; text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>

</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="1" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i><b>1</b> Introduction</a></li>
<li class="chapter" data-level="2" data-path="cv.html"><a href="cv.html"><i class="fa fa-check"></i><b>2</b> CV</a>
<ul>
<li class="chapter" data-level="" data-path="cv.html"><a href="cv.html#education"><i class="fa fa-check"></i>Education</a></li>
<li class="chapter" data-level="" data-path="cv.html"><a href="cv.html#important-courses"><i class="fa fa-check"></i>Important Courses</a></li>
<li class="chapter" data-level="" data-path="cv.html"><a href="cv.html#projects"><i class="fa fa-check"></i>Projects</a></li>
<li class="chapter" data-level="" data-path="cv.html"><a href="cv.html#minor-data-science-for-biology"><i class="fa fa-check"></i>Minor Data science for Biology</a></li>
<li class="chapter" data-level="" data-path="cv.html"><a href="cv.html#important-skills"><i class="fa fa-check"></i>Important Skills</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="guerilla-analytics.html"><a href="guerilla-analytics.html"><i class="fa fa-check"></i><b>3</b> <span style="color:#E656AE;">Guerilla Analytics</span></a></li>
<li class="chapter" data-level="4" data-path="reproducible-research.html"><a href="reproducible-research.html"><i class="fa fa-check"></i><b>4</b> <span style="color:#E656AE;">Reproducible Research</span></a>
<ul>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#c.-elegans-experiment"><i class="fa fa-check"></i>C. elegans experiment</a>
<ul>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#scatterplot"><i class="fa fa-check"></i>Scatterplot</a></li>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#normalizing-the-data"><i class="fa fa-check"></i>Normalizing the data</a></li>
</ul></li>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#article"><i class="fa fa-check"></i>Article</a>
<ul>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#summary"><i class="fa fa-check"></i>Summary</a></li>
<li class="chapter" data-level="" data-path="reproducible-research.html"><a href="reproducible-research.html#review"><i class="fa fa-check"></i>Review</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="new-packages.html"><a href="new-packages.html"><i class="fa fa-check"></i>New Packages</a>
<ul>
<li class="chapter" data-level="" data-path="new-packages.html"><a href="new-packages.html#visualisation-microplate-assay"><i class="fa fa-check"></i>Visualisation microplate assay</a></li>
<li class="chapter" data-level="" data-path="new-packages.html"><a href="new-packages.html#plater"><i class="fa fa-check"></i>plater</a></li>
<li class="chapter" data-level="" data-path="new-packages.html"><a href="new-packages.html#gganimate"><i class="fa fa-check"></i>gganimate</a></li>
<li class="chapter" data-level="4.1" data-path="new-packages.html"><a href="new-packages.html#geanimeerd-plot"><i class="fa fa-check"></i><b>4.1</b> <img src="animated_plot.gif" alt="Geanimeerd Plot" /></a></li>
<li class="chapter" data-level="4.2" data-path="new-packages.html"><a href="new-packages.html#qpcr"><i class="fa fa-check"></i><b>4.2</b> qPCR</a></li>
<li class="chapter" data-level="" data-path="new-packages.html"><a href="new-packages.html#references"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="sql.html"><a href="sql.html"><i class="fa fa-check"></i><b>5</b> SQL</a></li>
<li class="chapter" data-level="6" data-path="datetime-r-package.html"><a href="datetime-r-package.html"><i class="fa fa-check"></i><b>6</b> DateTime R Package</a>
<ul>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#introduction-1"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#installation"><i class="fa fa-check"></i>Installation</a></li>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#fuctions-of-the-package"><i class="fa fa-check"></i>Fuctions of the package</a>
<ul>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#calculating-date-difference"><i class="fa fa-check"></i>Calculating date difference</a></li>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#add-time-to-a-date"><i class="fa fa-check"></i>Add time to a date</a></li>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#check-if-two-dates-are-on-the-same-day"><i class="fa fa-check"></i>Check if two dates are on the same day</a></li>
<li class="chapter" data-level="" data-path="datetime-r-package.html"><a href="datetime-r-package.html#check-for-dutch-holidays"><i class="fa fa-check"></i>Check for Dutch holidays</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="projecticum-ner.html"><a href="projecticum-ner.html"><i class="fa fa-check"></i><b>7</b> Projecticum NER</a>
<ul>
<li class="chapter" data-level="" data-path="projecticum-ner.html"><a href="projecticum-ner.html#introduction-2"><i class="fa fa-check"></i>Introduction</a></li>
<li class="chapter" data-level="" data-path="projecticum-ner.html"><a href="projecticum-ner.html#physiologically-based-pharmacokinetic-models"><i class="fa fa-check"></i>Physiologically Based Pharmacokinetic models</a></li>
<li class="chapter" data-level="" data-path="projecticum-ner.html"><a href="projecticum-ner.html#prodigy"><i class="fa fa-check"></i>Prodigy</a></li>
<li class="chapter" data-level="" data-path="projecticum-ner.html"><a href="projecticum-ner.html#named-entity-recognition-models"><i class="fa fa-check"></i>Named Entity Recognition models</a></li>
<li class="chapter" data-level="" data-path="projecticum-ner.html"><a href="projecticum-ner.html#references-1"><i class="fa fa-check"></i>References</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="parameterized-covid-19-report.html"><a href="parameterized-covid-19-report.html"><i class="fa fa-check"></i><b>8</b> Parameterized COVID-19 report</a>
<ul>
<li class="chapter" data-level="" data-path="parameterized-covid-19-report.html"><a href="parameterized-covid-19-report.html#importing-and-inspecting"><i class="fa fa-check"></i>Importing and inspecting</a></li>
<li class="chapter" data-level="" data-path="parameterized-covid-19-report.html"><a href="parameterized-covid-19-report.html#line-chart-the-netherlands-belgium-and-germany"><i class="fa fa-check"></i>Line chart the Netherlands, Belgium and Germany</a></li>
<li class="chapter" data-level="" data-path="parameterized-covid-19-report.html"><a href="parameterized-covid-19-report.html#line-chart-romania-liechtenstein-and-italy"><i class="fa fa-check"></i>Line chart Romania, Liechtenstein and Italy</a></li>
</ul></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Portfolio</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="reproducible-research" class="section level1 hasAnchor" number="4">
<h1><span class="header-section-number">4</span> <span style="color:#E656AE;">Reproducible Research</span><a href="reproducible-research.html#reproducible-research" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="c.-elegans-experiment" class="section level2 unnumbered hasAnchor">
<h2>C. elegans experiment<a href="reproducible-research.html#c.-elegans-experiment" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p><em>The data for this exercise was kindly supplied by J. Louter (INT/ILC) and was derived from an experiment in which adult C.elegans nematodes were exposed to varying concentrations of different compounds.</em></p>
<p>First we load in the dataset.</p>
<div class="sourceCode" id="cb1"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb1-1"><a href="reproducible-research.html#cb1-1" tabindex="-1"></a><span class="fu">library</span>(readxl)</span>
<span id="cb1-2"><a href="reproducible-research.html#cb1-2" tabindex="-1"></a>CE_LIQ_FLOW_062_Tidydata <span class="ot">&lt;-</span> <span class="fu">read_excel</span>(<span class="st">&quot;~/CE.LIQ.FLOW.062_Tidydata.xlsx&quot;</span>)</span>
<span id="cb1-3"><a href="reproducible-research.html#cb1-3" tabindex="-1"></a><span class="fu">head</span>(CE_LIQ_FLOW_062_Tidydata)</span></code></pre></div>
<pre><code>## # A tibble: 6 × 34
##   plateRow plateColumn vialNr dropCode expType    expReplicate expName        
##   &lt;lgl&gt;    &lt;lgl&gt;        &lt;dbl&gt; &lt;chr&gt;    &lt;chr&gt;             &lt;dbl&gt; &lt;chr&gt;          
## 1 NA       NA               1 a        experiment            3 CE.LIQ.FLOW.062
## 2 NA       NA               1 b        experiment            3 CE.LIQ.FLOW.062
## 3 NA       NA               1 c        experiment            3 CE.LIQ.FLOW.062
## 4 NA       NA               1 d        experiment            3 CE.LIQ.FLOW.062
## 5 NA       NA               1 e        experiment            3 CE.LIQ.FLOW.062
## 6 NA       NA               2 a        experiment            3 CE.LIQ.FLOW.062
## # ℹ 27 more variables: expDate &lt;dttm&gt;, expResearcher &lt;chr&gt;, expTime &lt;dbl&gt;,
## #   expUnit &lt;chr&gt;, expVolumeCounted &lt;dbl&gt;, RawData &lt;dbl&gt;, compCASRN &lt;chr&gt;,
## #   compName &lt;chr&gt;, compConcentration &lt;chr&gt;, compUnit &lt;chr&gt;,
## #   compDelivery &lt;chr&gt;, compVehicle &lt;chr&gt;, elegansStrain &lt;chr&gt;,
## #   elegansInput &lt;dbl&gt;, bacterialStrain &lt;chr&gt;, bacterialTreatment &lt;chr&gt;,
## #   bacterialOD600 &lt;dbl&gt;, bacterialConcX &lt;dbl&gt;, bacterialVolume &lt;dbl&gt;,
## #   bacterialVolUnit &lt;chr&gt;, incubationVial &lt;chr&gt;, incubationVolume &lt;dbl&gt;, …</code></pre>
<p>You would expect the columns <span style="color:#E656AE;">RawData</span>, <span style="color:#E656AE;">compName</span> and <span style="color:#E656AE;">compConcentration</span> all to be numeric. These are the colums we are interested in.</p>
<ul>
<li><p><span style="color:#E656AE;">RawData</span> = the outcome - number of offspring counted as an integer value, after incubation time,</p></li>
<li><p><span style="color:#E656AE;">compName</span> = the generic name of the compound/chemical</p></li>
<li><p><span style="color:#E656AE;">compConcentration</span> = the concentration of the compound</p></li>
</ul>
<div class="sourceCode" id="cb3"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb3-1"><a href="reproducible-research.html#cb3-1" tabindex="-1"></a><span class="co"># inspect the data types of colums RawData, compName and compConcentration</span></span>
<span id="cb3-2"><a href="reproducible-research.html#cb3-2" tabindex="-1"></a><span class="fu">class</span>(CE_LIQ_FLOW_062_Tidydata<span class="sc">$</span>RawData)</span></code></pre></div>
<pre><code>## [1] &quot;numeric&quot;</code></pre>
<div class="sourceCode" id="cb5"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb5-1"><a href="reproducible-research.html#cb5-1" tabindex="-1"></a><span class="fu">class</span>(CE_LIQ_FLOW_062_Tidydata<span class="sc">$</span>compName)</span></code></pre></div>
<pre><code>## [1] &quot;character&quot;</code></pre>
<div class="sourceCode" id="cb7"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb7-1"><a href="reproducible-research.html#cb7-1" tabindex="-1"></a><span class="fu">class</span>(CE_LIQ_FLOW_062_Tidydata<span class="sc">$</span>compConcentration)</span></code></pre></div>
<pre><code>## [1] &quot;character&quot;</code></pre>
<p>But in R the variable <span style="color:#E656AE;">compConcentration</span> is not ‘numeric’, it is of the data type ‘character’.</p>
<div class="sourceCode" id="cb9"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb9-1"><a href="reproducible-research.html#cb9-1" tabindex="-1"></a><span class="co"># convert column compConcentration from character to numeric </span></span>
<span id="cb9-2"><a href="reproducible-research.html#cb9-2" tabindex="-1"></a><span class="co"># Remove NA and convert to numeric </span></span>
<span id="cb9-3"><a href="reproducible-research.html#cb9-3" tabindex="-1"></a><span class="fu">library</span>(dplyr)</span>
<span id="cb9-4"><a href="reproducible-research.html#cb9-4" tabindex="-1"></a></span>
<span id="cb9-5"><a href="reproducible-research.html#cb9-5" tabindex="-1"></a>CE_LIQ_FLOW_062_Tidydata <span class="ot">&lt;-</span> CE_LIQ_FLOW_062_Tidydata <span class="sc">%&gt;%</span></span>
<span id="cb9-6"><a href="reproducible-research.html#cb9-6" tabindex="-1"></a>  <span class="fu">mutate</span>(<span class="at">compConcentration =</span> <span class="fu">as.numeric</span>(compConcentration)) <span class="sc">%&gt;%</span> </span>
<span id="cb9-7"><a href="reproducible-research.html#cb9-7" tabindex="-1"></a>  <span class="fu">filter</span>(<span class="sc">!</span><span class="fu">is.na</span>(compConcentration))             </span></code></pre></div>
<div id="scatterplot" class="section level3 unnumbered hasAnchor">
<h3>Scatterplot<a href="reproducible-research.html#scatterplot" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Scatterplot of the data set for the different compounds and the varying concentrations down below:</p>
<p><img src="_main_files/figure-html/unnamed-chunk-4-1.png" width="672" /></p>
<p><span style="color:#E656AE;font-weight:bold;">Figuur 3. Scatterplot of different compounds and varying concentrations</span></p>
<p>The positive control for this experiment is <span style="color:#E656AE;">ethanol</span> and the negative control is <span style="color:#E656AE;">S-medium</span>.</p>
</div>
<div id="normalizing-the-data" class="section level3 unnumbered hasAnchor">
<h3>Normalizing the data<a href="reproducible-research.html#normalizing-the-data" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>For the dataset CE_LIQ_FLOW_062_Tidydata, the controleNegative condition was normalized so that its average value equals 1. All values are expressed as a fraction of this, which helps standardize the data and facilitates the interpretation and comparison of the effects of different conditions.</p>
<div class="sourceCode" id="cb10"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb10-1"><a href="reproducible-research.html#cb10-1" tabindex="-1"></a><span class="co"># Calculate mean value of controlNegative</span></span>
<span id="cb10-2"><a href="reproducible-research.html#cb10-2" tabindex="-1"></a>controlNegative <span class="ot">&lt;-</span> CE_LIQ_FLOW_062_Tidydata <span class="sc">%&gt;%</span> <span class="fu">filter</span>(CE_LIQ_FLOW_062_Tidydata<span class="sc">$</span>expType <span class="sc">==</span> <span class="st">&quot;controlNegative&quot;</span>)</span>
<span id="cb10-3"><a href="reproducible-research.html#cb10-3" tabindex="-1"></a>controlNegative_mean <span class="ot">&lt;-</span> <span class="fu">mean</span>(controlNegative<span class="sc">$</span>RawData) </span>
<span id="cb10-4"><a href="reproducible-research.html#cb10-4" tabindex="-1"></a>controlNegative_mean</span></code></pre></div>
<pre><code>## [1] 85.9</code></pre>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="reproducible-research.html#cb12-1" tabindex="-1"></a><span class="co"># Normalize data</span></span>
<span id="cb12-2"><a href="reproducible-research.html#cb12-2" tabindex="-1"></a>CE_LIQ_FLOW_062_Tidydata <span class="ot">&lt;-</span> CE_LIQ_FLOW_062_Tidydata <span class="sc">%&gt;%</span> <span class="fu">mutate</span>(<span class="at">normalized =</span> CE_LIQ_FLOW_062_Tidydata<span class="sc">$</span>RawData <span class="sc">/</span> controlNegative_mean)</span>
<span id="cb12-3"><a href="reproducible-research.html#cb12-3" tabindex="-1"></a></span>
<span id="cb12-4"><a href="reproducible-research.html#cb12-4" tabindex="-1"></a><span class="co"># Inspect the data types of columns RawData, compName, expType, compConcentration and normalized</span></span>
<span id="cb12-5"><a href="reproducible-research.html#cb12-5" tabindex="-1"></a>CE_LIQ_FLOW_062_Tidydata <span class="sc">%&gt;%</span> <span class="fu">select</span>(RawData, compName, compConcentration, expType, normalized)</span></code></pre></div>
<pre><code>## # A tibble: 359 × 5
##    RawData compName                   compConcentration expType    normalized
##      &lt;dbl&gt; &lt;chr&gt;                                  &lt;dbl&gt; &lt;chr&gt;           &lt;dbl&gt;
##  1      44 2,6-diisopropylnaphthalene              4.99 experiment      0.512
##  2      37 2,6-diisopropylnaphthalene              4.99 experiment      0.431
##  3      45 2,6-diisopropylnaphthalene              4.99 experiment      0.524
##  4      47 2,6-diisopropylnaphthalene              4.99 experiment      0.547
##  5      41 2,6-diisopropylnaphthalene              4.99 experiment      0.477
##  6      35 2,6-diisopropylnaphthalene              4.99 experiment      0.407
##  7      41 2,6-diisopropylnaphthalene              4.99 experiment      0.477
##  8      36 2,6-diisopropylnaphthalene              4.99 experiment      0.419
##  9      40 2,6-diisopropylnaphthalene              4.99 experiment      0.466
## 10      38 2,6-diisopropylnaphthalene              4.99 experiment      0.442
## # ℹ 349 more rows</code></pre>
<p><img src="_main_files/figure-html/unnamed-chunk-6-1.png" width="672" /></p>
<p><span style="color:#E656AE;font-weight:bold;">Figuur 4. Scatterplot of different compounds and varying concentrations normalized with the mean of the negative control</span></p>
<hr />
</div>
</div>
<div id="article" class="section level2 unnumbered hasAnchor">
<h2>Article<a href="reproducible-research.html#article" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>An, A.Y. et al. (2023) Dynamic gene expression analysis reveals distinct severity phases of immune and cellular dysregulation in COVID-19, bioRxiv. Available at: <a href="https://www.biorxiv.org/content/10.1101/2023.11.04.565404v1.full" class="uri">https://www.biorxiv.org/content/10.1101/2023.11.04.565404v1.full</a> (Accessed: 02 May 2024).</p>
<div id="summary" class="section level3 unnumbered hasAnchor">
<h3>Summary<a href="reproducible-research.html#summary" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>Objective
To investigate longitudinal changes in gene expression profiles throughout the COVID-19 disease timeline.</p>
<p>Methods
Three-hundred whole blood samples from 128 adult patients were collected during hospitalization from COVID-19, with up to five samples per patient. Transcriptome sequencing (RNA-Seq), differential gene expression analysis and pathway enrichment was performed. Drug-gene set enrichment analysis was used to identify FDA-approved medications that could inhibit critical genes and proteins at each disease phase. Prognostic gene-expression signatures were generated using machine learning to distinguish 3 disease stages.</p>
<p>Results
Samples were longitudinally grouped by clinical criteria and gene expression into six disease phases: Mild, Moderate, Severe, Critical, Recovery, and Discharge. Distinct mechanisms with differing trajectories during COVID-19 hospitalization were apparent. Antiviral responses peaked early in COVID-19, while heme metabolism pathways became active much later during disease. Adaptive immune dysfunction, inflammation, and metabolic derangements were most pronounced during phases with higher disease severity, while hemostatic abnormalities were elevated early and persisted throughout the disease course. Drug-gene set enrichment analysis predicted repurposed medications for potential use, including platelet inhibitors in early disease, antidiabetic medications for patients with increased disease severity, and dasatinib throughout the disease course. Disease phases could be categorized using specific gene signatures for prognosis and treatment selection. Disease phases were also highly correlated to previously developed sepsis endotypes, indicating that severity and disease timing were significant contributors to heterogeneity observed in sepsis and COVID-19.</p>
<p>Conclusions
Higher temporal resolution of longitudinal mechanisms in COVID-19 revealed multiple immune and cellular changes that were activated at different phases of COVID-19. Understanding how a patient’s gene expression profile changes over time can permit more accurate risk stratification of patients and provide time-dependent personalized treatments with repurposed medications. This creates an opportunity for timely intervention before patients transition to a more severe phase, potentially accelerating patients to recovery.</p>
</div>
<div id="review" class="section level3 unnumbered hasAnchor">
<h3>Review<a href="reproducible-research.html#review" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<p>The article has been evaluated according to the criteria listed in the table below.</p>
<table>
<colgroup>
<col width="6%" />
<col width="57%" />
<col width="34%" />
<col width="0%" />
<col width="0%" />
</colgroup>
<thead>
<tr class="header">
<th align="center">Transparency Criteria</th>
<th align="center">Definition</th>
<th align="center">Response</th>
<th></th>
<th></th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td align="center">Study Purpose</td>
<td align="center">A concise statement in the introduction of the article, often in the last paragraph, that establishes the reason the research was conducted. Also called the study objective.</td>
<td align="center">yes</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td align="center">Data Availability Statement</td>
<td align="center">A statement, in an individual section offset from the main body of text, that explains how or if one can access a study’s data. The title of the section may vary, but it must explicitly mention data; it is therefore distinct from a supplementary materials section.</td>
<td align="center">yes</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td align="center">Data Location</td>
<td align="center">Where the article’s data can be accessed, either raw or processed.</td>
<td align="center">The datasets generated for this study can be found on GEO: GSE221234 and GSE222253. Code is available upon request (Dr. Robert Hancock, bob{at}hancocklab.com).</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td align="center">Study Location</td>
<td align="center">Author has stated in the methods section where the study took place or the data’s country/region of origin.</td>
<td align="center">yes,</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td align="center">Author Review</td>
<td align="center">The professionalism of the contact information that the author has provided in the manuscript.</td>
<td align="center">yes, with information of the researchers and an e-mail.</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td align="center">Ethics Statement</td>
<td align="center">A statement within the manuscript indicating any ethical concerns, including the presence of sensitive data.</td>
<td align="center">yes</td>
<td></td>
<td></td>
</tr>
<tr class="odd">
<td align="center">Funding Statement</td>
<td align="center">A statement within the manuscript indicating whether or not the authors received funding for their research.</td>
<td align="center">yes</td>
<td></td>
<td></td>
</tr>
<tr class="even">
<td align="center">Code Availability</td>
<td align="center">Authors have shared access to the most updated code that they used in their study, including code used for analysis.</td>
<td align="center">no</td>
<td></td>
<td></td>
</tr>
</tbody>
</table>

</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="guerilla-analytics.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="new-packages.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": null,
"text": null
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

</body>

</html>
